### DISCUSSION TITLE
Unable to query all service logs by traceId in SkyWalking UI 9.4.0

### DISCUSSION THREAD

**Discussion by kevin-nannan** [apache/skywalking#11040] [Q&A] [answered]
Description:
    I am using SkyWalking UI 9.4.0. However, I have encountered a limitation where I am unable to query all service-related logs based on a specific TraceID. Currently, I can only search for logs associated with a particular TraceID within a specific service.
    However, I have discovered that the UI in version 8.7.0 does support querying a TraceID across all services.
    I would like to know whether it is a configuration issue on my end or if there is a specific reason why this feature was removed.
描述:
  我正在使用skywalking UI 9.4.0。但是我发现根据traceId查询日志时，只能选择某一服务。但是我观察之前8版本的UI是支持all services选中的。我想知道是我配置问题还是由于什么原因取消了这个功能

**Comment by wu-sheng** [comment, marked as answer]
We prefer you don't do that. It is easy to overhead ElasticSearch server.
Maybe when BanyanDB option ready for product, we would reopen that.

FYI @hanahmily for a use case of new database.

**Reply by jampsen** [reply]
Thank you for your response, and we are facing the same problem. 
In our real-world microservices scenarios, a TraceId spans across multiple Service and correlates with Logs. When troubleshooting, we want to view all logs associated with a TraceId on a single page. 
However, after upgrading to 9.x, each page requires selecting a Service, and when navigating from Trace to Log page, we need to manually switch between multiple services to achieve logical correlation. Additionally, for some developer, remembering the exact names of each microservice can be challenging. 
If performance considerations for Elasticsearch led to this design, we believe that users should have the option to make this choice themselves.
Like 8.X UI
![image](https://github.com/apache/skywalking/assets/35960370/9da5fb8c-93ed-442b-9bed-7d9fe320426b)

**Reply by wu-sheng** [reply]
We would not provide that. If you want to change, a private fork is better.
